sparsimony/utils.py

```diff
@@ -74,3 +74,21 @@ def share_parametrizations(
         primary_para.replicas_.append(replica_para)
     else:
         primary_para.replicas_ = [replica_para]
+
+
+def calculate_n_drop(mask: torch.Tensor, target_sparsity: float) -> int:
```

Comment:
let's just call the argument sparsity
```suggestion
def calculate_n_drop(mask: torch.Tensor, sparsity: float) -> int:
```

Reply:
Done